unittests/InstructionCountCI/AFP/VEX_map1.json

```diff
@@ -39,7 +41,8 @@
         "mov v16.16b, v17.16b",
         "fmov s0, #0x70 (1.0000)",
         "fsqrt s1, s18",
-        "fdiv s16, s0, s1"
+        "fdiv s0, s0, s1",
```

Comment:
This one looks like it was a legitimate problem.
Pseudocode:
```
boolean merge = IsMerging(FPCR);
bits(128) result = if merge then V[n, 128] else Zeros(128);
```
Since vn here is s0 which contains 1.0, it wasn't doing correct insert mechanics. AFP+RPRES would be necessary to get correct semantics.
:+1: